api/rest/v1/bulk_find_entities.md

```diff
@@ -0,0 +1,157 @@
+---
+layout: default
+title: Find DCIDs
```

Comment:
Do you think we can call this "Bulk Find DCIDs"? Or something else to distinguish between /bulk/find/entities and /find/entities?

Reply:
Right now we give simple and bulk pages the same title so it doesn't overwhelm the sidebar. We should consider rearranging so both simple and bulk endpoints are linked... I'll send a separate PR for that update.